src/main/antora/modules/ROOT/pages/aot.adoc

```diff
@@ -32,30 +37,110 @@ However, there users may fine tune the configuration with following options.
 
 |`spring.aot.data.accessors.include`
 |Comma separated list of FQCN for which to contribute Bytecode for generated Type/Property Accessors.
-Ant-style include patterns matching package names (e.g. `com.acme.**`) or type names inclusion.
+Ant-style include patterns matching package names (e.g. `example.springdata.**`) or type names inclusion.
 Inclusion pattern matches are evaluated before exclusions for broad exclusion and selective inclusion.
```

Comment:
I find this order of evaluation rather confusing, because it is really about precedence. 

Something like this seems easier to understand.

If a type is matched by a inclusion and an exclusion, the exclusion _wins_ and the type is considered excluded.